src/components/projectRegister/ValidationModal.tsx

```
@@ -0,0 +1,293 @@
+
+        {/* 1단계 항목들 */}
+        <div className="mb-6">
+          <h3 className="text-lg font-semibold mb-3 text-main-color">1단계 - 기본 정보</h3>
+          <div className="space-y-2">
+            {step1Items.map((item, index) => (
+              <div key={index} className="flex items-center justify-between py-2">
+                <span className="text-sm">{item.label}</span>
+                <div className="flex items-center">
+                  {item.isValid ? (
+                    <div className="flex items-center text-green-600">
+                      <Check className="w-4 h-4 mr-1" />
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)